src/main/java/io/kokuwa/maven/helm/UpgradeMojo.java

```diff
@@ -0,0 +1,45 @@
+package io.kokuwa.maven.helm;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import lombok.Setter;
+
+/**
+ * Mojo for simulate a Upgrade.
+ */
+@Mojo(name = "upgrade", defaultPhase = LifecyclePhase.DEPLOY, threadSafe = true)
+@Setter
+public class UpgradeMojo extends AbstractHelmWithValueOverrideMojo {
+
```

Comment:
I would rather skip it on default. Just to evade the risk of someone installing/upgrading something on an actual cluster without beeing aware of it.

Reply:
changed